test/erc-20-extensions/ERC20Extensions.js

```diff
@@ -252,3 +254,49 @@ describe('ERC20ExtensionsMock tests', function () {
     })
   })
 })
+
+// Transaction needs to be propagated to the mirror node
+async function pollForChangedSupply(ERC20Burnable, initialSupply){
+  let numberOfTries = 0;
+  const timesToTry = 300;
```

Comment:
This is an alarmingly high number of tries.
Looking down the delay is 4 secs. SO this could go for 1200 secs (20mins) if the scenarios aligned.
I think you shouldn't need more than 2 tries if the delay is 4 secs (2 block times)

What the average number of retries you're seeing?